numpy/core/getlimits.py

```diff
@@ -343,6 +360,12 @@ class finfo:
     tiny : float
         The smallest positive floating point number with full precision
         (see Notes).
```

Comment:
How about we replace this with "An alias for `smallest_normal`, kept for backwards compatibility"

Reply:
This references the `Notes` section - that needs referencing from `smallest_normal` instead, and then updating that text (e.g. replace `tiny` there).